include/config/rcxml.h

```diff
@@ -69,6 +69,7 @@ struct rcxml {
 
 	struct {
 		int popuptime;
+		int count;
```

Comment:
I think we should find a more descriptive variable name here. “count” is a bit like “i” which is fine in a loop where it’s obvious what it does.

I’m on a phone so can’t easily navigate around the patch, but think I’m right in thinking that “count” is only used in post_processing().

So, does the variable specify the minimum number of workspaces desired by user?

Could we call it ‘min_nr_workspaces’ ? I think that would be helpful to people looking at this in the future. And easier to grep for.

We have tended to use ‘nr_’ as a “number of” prefix throughout the code base rather than “num_” by the way.

Reply:
> I’m on a phone so can’t easily navigate around the patch, but think I’m right in thinking that “count” is only used in post_processing().

It is used in entry() as well.

> So, does the variable specify the minimum number of workspaces desired by user?

Yes that is correct!

> Could we call it ‘min_nr_workspaces’ ? I think that would be helpful to people looking at this in the future. And easier to grep for.
> 
> We have tended to use ‘nr_’ as a “number of” prefix throughout the code base rather than “num_” by the way.

Noted.